spec/support/doc_auth_image_fixtures.rb

```diff
@@ -98,4 +98,33 @@ def self.fixture_path(filename)
   def self.load_image_data(filename)
     File.read(fixture_path(filename))
   end
+
+  def self.zipped_files(reference_id:, selfie: false, passport: false)
+    temp_dir = Dir.mktmpdir(reference_id)
+
+    if passport
+      # i don't think this is possible with socure?
```

Comment:
i was told that passports are on our roadmap with socure, so i am going to leave this in